README.md

```diff
@@ -74,7 +74,6 @@ A demo of the DevOps Platform is on the [website's landing page](https://sphinx.
 ## Current Limitations
 
 - You cannot deploy [libraries](https://docs.soliditylang.org/en/v0.8.24/contracts.html#libraries).
-- You cannot send ETH as part of a deployment.
```

Comment:
Context: In the previous PR, you left a [comment](https://github.com/sphinx-labs/sphinx/pull/1403#discussion_r1477545328) saying that we should allow transferring ETH from the Safe (as long as the Safe is funded). We now allow this.

I think we should remove this sentence from the "Current Limitations" section because we lifted the restriction. I figured it'd be somewhat arbitrary to say "We don't airdrop tokens to your Safe" because it's a random feature that nobody's requested

Reply:
I agree, no one expects airdrops to be supported.